builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -771,7 +792,6 @@ def unDeployWebsite(stage) {
 					sh "aws s3 rm s3://${assetInfo['folder_name']} --recursive"
 				}
 			}
-			resetCredentials(credsId)
```

Comment:
resetCredentials should be in `finally` block.

Reply:
done